File: Swift/Strings/ReverseString.swift
import Foundation 

func reverseString(_ s: String) -> String {
    var chars = Array(s)
    var left = 0, right = chars.count - 1
    while left < right {
        chars.swapAt(left, right)
        left += 1
        right -= 1
    }
    return String(chars)
}
print(reverseString("hello")) // "olleh"

File: Python/Strings/reverseString.py

# def reverse(s: str)->str:
#     reversedString = ''
#     for char in s:
#         reversedString = char+reversedString
#     return reversedString
# print(reverse('hello'))


#string slicing 
def reverse_string(s: str) -> str:
    return s[::-1]
string = 'hello'
print(reverse_string(string))




